pyjexl/parser.py

```diff
@@ -253,14 +253,15 @@ def __new__(meta, classname, bases, classdict):
         return type.__new__(meta, classname, bases, classdict)
 
 
-class Node(object, metaclass=NodeMeta):
+class Node(object):
     """
     Base class for AST Nodes.
 
     Nodes are like mutable namedtuples. Each node type should declare a
     fields attribute on the class that lists the desired attributes for
     the class.
     """
+    __metaclass__ = NodeMeta
```

Comment:
I think this won't work on Python 3? Either way, future has a [`with_metaclass`](http://python-future.org/compatible_idioms.html#metaclasses) helper for this.